generators/app/index.js

```diff
@@ -2,8 +2,12 @@ var Generator = require('yeoman-generator')
 var chalk = require('chalk')
 var yosay = require('yosay')
 
-const COMPONENT = 'Components'
-const SCENE = 'Scenes'
+const CLASS_COMPONENT = 'Class Component'
+const CLASS_SCENE = 'Class Scene'
+const FUNCTIONAL_COMPONENT = 'Functional Component'
+const FUNCTIONAL_SCENE = 'Functional Scene'
```

Comment:
Much as I keep referring to them as functional components, the docs call them 'function components', and I think  we should probably do the same https://reactjs.org/docs/components-and-props.html.

Functional component may infer some functional programming concepts that aren't going to be true of all function components. (I got sent this the other day which does a much better job of explaining it than I would https://dev.to/jackmellis/the-slightly-misleading-terminology-of-react-hooks-la5)